### ISSUE TITLE
Validation errors are display on modals or slideover

### ISSUE THREAD

**Issue by richardkeep** [protonemedia/laravel-splade#7]
Thanks for the wonderful package, Pascal. I am really liking it.

Create a simple form with a `<x-splade-form>` and wrap it in a `<x-splade-modal>` component. Load the page via `<Link modal href="...">`. Ensure you have set up the Laravel Validation Rules in a controller method.

The form will submit and you can see the validation errors on the *Network* tab. However, neither of the 2 options will display errors on the page.
```html
<p v-text="form.errors.name" />
```
or
```html
<x-splade-errors>
    <p v-if="errors.has('name')" v-text="errors.first('name')" />
</x-splade-errors>
```

When you remove the `<x-splade-modal>` wrapper, the validation errors are displayed on the page as expected.

**Comment by pascalbaljet** [comment]
Thanks for sharing! I've built it so that the validation errors are available/shown in the modal, not in the 'parent' below. I'll investigate!

**Comment by richardkeep** [comment]
Hmm, just tried again using the [changes](https://github.com/protonemedia/laravel-splade/commit/9e85a3b335b105b37d6a38bc3f997613be50fd56) but the problem still persists for both modal and slideover.

**Comment by pascalbaljet** [comment]
Fixed in v0.1.2. Be sure to update both with composer and npm.

**Comment by richardkeep** [comment]
Works like charm! Thanks
